platform/src/main/java/org/stellar/anchor/platform/configurator/ConfigEnvironment.java

```diff
@@ -46,7 +46,11 @@ public static void rebuild() {
    * @return the value of the environment variable.
    */
   public static String getenv(String name) {
-    return env.get(toPosixForm(name));
+    String envValue = env.get(toPosixForm(name));
+    if (envValue != null) {
+      envValue = envValue.replace("\\n", "\n").replace("\\\"", "\"");
```

Comment:
I am a little worried about the corner cases that we did not cover such as "\\t" for tab.

Can we use [`StringEscapeUtils` from Apache](https://commons.apache.org/proper/commons-text/apidocs/org/apache/commons/text/StringEscapeUtils.html#unescapeJava-java.lang.String-)?

Reply:
Great suggestion, thank you! Done in https://github.com/stellar/java-stellar-anchor-sdk/pull/824/commits/4f2cdfa9a36837d7d16ca34a7b36bd7ca630c5d6.